File: src/main/java/in/dragonbra/javasteam/steam/steamclient/callbackmgr/ICallbackMsg.java
package in.dragonbra.javasteam.steam.steamclient.callbackmgr;

import in.dragonbra.javasteam.types.JobID;

/**
 * A callback message
 */
public interface ICallbackMsg {
    JobID getJobID();

    void setJobID(JobID jobID);
}

File: src/main/java/in/dragonbra/javasteam/steam/steamclient/callbackmgr/ICallbackMsg.kt
package `in`.dragonbra.javasteam.steam.steamclient.callbackmgr

import `in`.dragonbra.javasteam.types.JobID

/**
 * A callback message
 */
interface ICallbackMsg {

    /**
     * The [JobID] that this callback is associated with. If there is no job associated,
     * then this will be [JobID.INVALID]
     */
    var jobID: JobID
}
